### ISSUE TITLE
HOW discussion: How can ACCELNET support collaboration to close knowledge gaps?

### ISSUE THREAD

**Issue by crossley** [crossley/ARCANI#4]
Please share your feedback on this group discussion by replying in this thread. Your contributions are important as we aim to identify gaps in knowledge, pinpoint collaborators who can help us bridge these gaps, and explore how sustainable collaboration within the ARCANI network can address these issues. We will incorporate the thoughts and opinions expressed here into a white paper focused on these critical questions.

**Comment by Daspraelon** [comment]
The total grant is likely to be about $2M. Once you take away all the nonsense, that doesn’t leave a ton of money for human time. What do you think are efficient uses of money that maximize networking successes?

**Comment by HoverflyLab** [comment]
Mobility funding, for e.g. students to exchange techniques and knowledge. Longer visits are important.

**Comment by BrunovanSwinderen** [comment]
Supporting higher degree research students is crucial. They are the ones often doing most of the work (e.g., experiments) so will benefit most from collaborative stays with colleagues, to learn new techniques, etc. Also, they find this very rewarding and it often motivates them to stay in the field and find postdocs in collaborating labs. This promotes continuity for projects that often take lots of time.

**Comment by ABBarron** [comment]
> The total grant is likely to be about $2M. Once you take away all the nonsense, that doesn’t leave a ton of money for human time. What do you think are efficient uses of money that maximize networking successes?

yes indeed - and it will be 2M over 3 years if we get it.

**Comment by ABBarron** [comment]
ECRs!!!!!! This kind of networking and connection can give a huge uplift to an ECR trajectory.  Plus ECRs can leverage the involvement for their own grant and paper successes.

**Comment by HoverflyLab** [comment]
There's a benefit in meeting face to face, but online meetings work when social connections have already been made (and saves travelling time and time away from family and other commitments)

**Comment by patrickmcgivern** [comment]
Having resources to share experiences relating to visiting labs, engaging in workshops, etc. would be helpful especially for ECRs - something to provide practical suggestions on how to make the most of such opportunities

**Comment by kozzy97** [comment]
To free time for the PIs, it seems like a good use could be to hire support the administrative side of organising the network, the workshops, funding applications, etc.

**Comment by ABBarron** [comment]
cross-disciplinary summer schools. This came up in the WHAT conversation but it would be amazing to have a summer school where an old neuroscientist like me could learn machine learning theory and machine learning theorists could learn principles of neuroscience.

**Comment by ABBarron** [comment]
> Supporting higher degree research students is crucial. They are the ones often doing most of the work (e.g., experiments) so will benefit most from collaborative stays with colleagues, to learn new techniques, etc. Also, they find this very rewarding and it often motivates them to stay in the field and find postdocs in collaborating labs. This promotes continuity for projects that often take lots of time.

yes - a specific visit for a month or so to upskill in a new field could be hugely beneficial for a PhD student

**Comment by Somayeh-h** [comment]
Face to face meetings (in small groups, with discussions for a short term) are much better in terms of meeting with people, and have engaging discussions. Online meetings makes it harder to have great discussions, and hard to focus, but if we know most people in the online meeting, then that’s much better.

**Comment by ookwrd** [comment]
We talked about the concept of « sticky campus », designed to keep students on site, interacting in person. There must be features of design for meetings that make for richer interactions.

**Comment by ABBarron** [comment]
Retreats in pretty places away from a home institution.  I've found mountainous isolated 3-day retreats with wifi but not the demands of campus can be hugely productive for long conversations and getting writing done.

**Comment by jarmarshall** [comment]
Table discussion - @Daspraelon - convene 'fringe' events at mainstream conferences, e.g. NeurIPS to bring together researchers who would not usually attend

**Comment by ookwrd** [comment]
Design general good practice, for ARCANI to support parents (especially with young children) to be able to participate and engage in meetings.

**Comment by Somayeh-h** [comment]
Maybe now that we have the option on doing teaching online, it makes it more possible to travel, considering other commitments.

**Comment by kozzy97** [comment]
Smaller, more frequent workshops for particular geographical regions of the network

**Comment by Somayeh-h** [comment]
Event duration should not be long enough to impact regular schedule, but in short term like this workshop, it’s great to have this time just to focus on discussions.

**Comment by RMenary** [comment]
PhD, post-doc exchange programme.

**Comment by BrunovanSwinderen** [comment]
The most productive collaborations are with colleagues who have become your friends over time. True collaborations emerge mostly from bottom-up interactions like that, rather than top-down plans. Friends are the ones you share crazy ideas with, you're not afraid of looking dumb, you trust them. How you become friends with other scientists is all about breaking bread with them. It takes time, and doesn't have to be all about the science to begin with. Sounds elitist to find ways to support this more, but it really does work to promote creativity.

**Comment by HoverflyLab** [comment]
Collective list of conferences that people re going to anyway, to set yp face-to-face meetings or organize workshops

**Comment by RMenary** [comment]
More opportunities for adventurous thinking!

**Comment by ABBarron** [comment]
> Smaller, more frequent workshops for particular geographical regions of the network

yes - smaller regular meetings with less of a travel burden and fewer larger collective meetings.  So a fractal type model

**Comment by ookwrd** [comment]
Share a list of conferences and events we are attending – so as to help decision making in terms of selecting events and facilitating the organization of workshops on topics of interest at those events.

**Comment by alexjgillett1984** [comment]
Regular workshops would be good but need to be targetted. Workshops, working parties, and exchanges between labs should be orientated towards some kind of permanent network that needs to be anchored to a "something" in an institution

**Comment by HoverflyLab** [comment]
Non-traditional discussions - e.g. while hiking, or doing some other physical activity (assuming ablebody-ness)

**Comment by ABBarron** [comment]
> Regular workshops would be good but need to be targetted. Workshops, working parties, and exchanges between labs should be orientated towards some kind of permanent network that needs to be anchored to a "something" in an institution

yeah we do need an anchor to keep things moving and keep an eye on overall progress. Plus there will be a non-trivial admin burden to this.

**Comment by ookwrd** [comment]
Co-doodling (without further comments for now)

**Comment by kozzy97** [comment]
ARCANI discord server (as a 'bespoke digital resource')

**Comment by alexjgillett1984** [comment]
Exchanges between labs: Lots of value for bringing in people from differing backgrounds/perspectives. Dissemination and communication of ideas. Being able to bring in someone with expertise in an area that may be lacking in the lab to run seminars for postdocs and grad students, (etc.). Interdisciplinary exchanges are what are most important here. Focus particularly on travel grants for ECRs and post-docs

**Comment by crossley** [comment]
I will flush these out when I get a chance, but here is the bullet list of our discussion points:

- Ultimately not a lot of money

- residency
- zoom
- conference

- write out Kalman Filter Wolpert / Ivry story
- RL / conditioning examples
-- Bob Taylor
-- Leonard Kleinrock
-- Joseph Licklider
-- Larry Roberts

- Meeting of a small group of elites

- administrative resources / designated grant write

**Comment by Daspraelon** [comment]
Grant writer/funding sourcer

**Comment by Daspraelon** [comment]
Family-friendly designs of events/funding opportunities.

**Comment by HoverflyLab** [comment]
Time saving: Use machine learning to speed up some training tasks (i.e. identifying individual monkeys). Also creates collaborations between fields (e.g. primatology and machine learning).

**Comment by HoverflyLab** [comment]
> Family-friendly designs of events/funding opportunities.

As well as having other cultural obligations.

**Comment by alexjgillett1984** [comment]
Longer visits – summer/winter school. Focus on bringing in ECRs and grad/post-grads. Not just seminars and workshops. But also involving non-academic activities (e.g., rock climbing, snorkelling, bushwalking, etc.). Would have group projects that push people to be interdisciplinary.

**Comment by Daspraelon** [comment]
Facilitated information regarding who’s going to what sort of meeting when…

**Comment by gmfricke** [comment]
I like an idea presented at our table of hiring a grant support person funded to save researcher time by understanding the funding landscape and identifying and supporting co-PIs who fit the bill during the grant submission process. This would address how ARCANI can save researcher time. Drawback that was pointed out is the delay in seeing returns.

**Comment by ABBarron** [comment]
> I like an idea presented at our table of hiring a grant support person funded to save researcher time by understanding the funding landscape and identifying and supporting co-PIs who fit the bill during the grant submission process. This would address how ACARIS can save researcher time. Drawback that was pointed out is the delay in seeing returns.

this could be really important because to source funding that can be used across countries might mean tapping into funding pots that are not conventionally used.

**Comment by HoverflyLab** [comment]
Interdisciplinary, intercontinental job platform with upcoming opportunities . This could free up time for ECRs (who spend a lot of time worrying about where there salary is coming from next) to instead focus on the next crazy idea.

**Comment by ookwrd** [comment]
Investing more into local conferences? Cheaper for travel and can be higher quality/food/comfort – motivating to free time.

**Comment by ookwrd** [comment]
Building a map of people + events.

**Comment by MartaHalina** [comment]
A prize to reward interdisciplinary work and help overcome disciplinary silos.

**Comment by RMenary** [comment]
project/problem based meetings to develop publications and grant applications - Somayeh's idea.

**Comment by ookwrd** [comment]
> A prize to reward interdisciplinary work and help overcome disciplinary silos.

Excellent idea. Perhaps some award(s) as well, if there is a structure/committee/society giving them out?

**Comment by BrunovanSwinderen** [comment]
Promote more local gatherings that are well-supported, rather than spending on airfares. We often don't know who's doing related work in our area. We end up finding this out by meeting in faraway places. We could use a local network map to identify interested parties. Then support these with gatherings that aren't necessarily about the science to begin with, but just break ice and bread. Use the saved airfare costs for interesting settings. This could help establish the kind of friendship/collaborations that are more sustainable through time because they're local.

**Comment by ABBarron** [comment]
> The most productive collaborations are with colleagues who have become your friends over time. True collaborations emerge mostly from bottom-up interactions like that, rather than top-down plans. Friends are the ones you share crazy ideas with, you're not afraid of looking dumb, you trust them. How you become friends with other scientists is all about breaking bread with them. It takes time, and doesn't have to be all about the science to begin with. Sounds elitist to find ways to support this more, but it really does work to promote creativity.

yes - and are there activities that we could put in place that could foster that sense of trust and build friendship in the community? Social engineering is rarely successful, but if this could be done we might achieve that goal of an enduring network that outlives the life of the grant.

**Comment by ABBarron** [comment]
> project/problem based meetings to develop publications and grant applications.

yes!  So meetings become productive and catalysers of more inputs and outputs to the network. That could help the network move along to "sustainability" beyond the life of the grant.

**Comment by jarmarshall** [comment]
Break down interdisciplinary communication boundaries through a wiki that crowd-sources definitions, allows disambiguation on terminology, definitions of fields (e.g. neuro-anthropology) - project officer with a training in one relevant science background could curate

**Comment by ABBarron** [comment]
> > A prize to reward interdisciplinary work and help overcome disciplinary silos.
> 
> Excellent idea. Perhaps some award(s) as well, if there is a structure/committee/society giving them out?

ooooh!  A prize!  This could be totally transformative for an ECR trajectory but it would not cost much!

**Comment by ookwrd** [comment]
Pairing a post-summer school grant writing activity, with some senior people joining/advising the writing – can be very efficient because there is a shared language and motivation and time to think and write.

**Comment by Somayeh-h** [comment]
Workshops like the Telluride Neuromorphic workshop which is a combination of talks (for the first week) then working towards a project for the next three weeks, and doing activities outside such as hiking to help having discussions in different settings would be great. Link: https://sites.google.com/view/telluride-2024/home?authuser=0

**Comment by ookwrd** [comment]
More cheese (serious idea about science backchanneling – Bruno or Pierrick can explain)

**Comment by HoverflyLab** [comment]
Follow-up meeting (could be online) to share how people have used the knowledge gained/connections made - forces you to not just get immediately submerged by your every day job.

**Comment by Daspraelon** [comment]
What team building would really work?

**Comment by ABBarron** [comment]
> Promote more local gatherings that are well-supported, rather than spending on airfares. We often don't know who's doing related work in our area. We end up finding this out by meeting in faraway places. We could use a local network map to identify interested parties. Then support these with gatherings that aren't necessarily about the science to begin with, but just break ice and bread. Use the saved airfare costs for interesting settings. This could help establish the kind of friendship/collaborations that are more sustainable through time because they're local.

Yes! This is coming up! A global-local model so we can keep activity running locally and less frequently pull a larger team together.

**Comment by HoverflyLab** [comment]
we need more Andy Barrons (amazing ability to see connections where others see silos)

**Comment by gmfricke** [comment]
Need to fund LLM Andrew emulator to make interdisciplinary connections.

**Comment by stenti** [comment]
Inviting funders to meetings

**Comment by jarmarshall** [comment]
Invite funders to meetings (Virginia)

**Comment by ABBarron** [comment]
models of successful collaborative networks
Mathematical Biosciences networks - NSF funded
Marie Curie training networks
NESCENT - 3 meetings over 18 months?
global scholarships - split across institutions

**Comment by ABBarron** [comment]
supporting ECRs  - TWCF priority. ECR co-uplift.  There is more value to ECRs promoting each other than a mentor doing it. More sustained impact and ongoing ECR support.

**Comment by ABBarron** [comment]
mentoring of assoc prof level - helping a new leader establish.  Buy out a course and help a new emerging person get their first grant to get their wings up. Catalyse. A regranting scheme - give an emerging leader a capacity to seed small projects and lead their own network.

**Comment by ABBarron** [comment]
WRITE CATAYLYTIC AND HIGH RISK HIGH REWARD INTO GRANTS. Booyah!

**Comment by ABBarron** [comment]
DI and other communities needs to fledge independent of the initial funding sources.

**Comment by fgashby** [comment]
At UC Santa Barbara, we started an interdisciplinary PhD program about 15 years ago that we called Dynamical Neuroscience. That program, which still exists, includes faculty from 7 or 8 campus departments and is broad enough to house all of the research described at this workshop under its umbrella. One of our greatest challenges though was to fund graduate students. This was largely because we used existing campus coursework to build the program, and the departments that offered those courses controlled the course TA assignments. For this reason, we didn't have any TAs to offer as funding for our grad students. Instead, all students were funded via fellowship or individual faculty grants. It would have made a huge difference if we could have funded even one student a year via some kind of ARCANI fellowship. A viable program requires enough students to form a  cohort, and especially in the beginning, we were barely able to do this. Even one more student a year would have been an enormous advantage to the whole program.

**Comment by crossley** [comment]
Ultimately may not be a lot of money. This was a key idea to emerge from my table 1 discussions. The consequence of this may be that many of our more traditional ideas -- e.g., longer term residential lab visits, postdoc exchanges, etc -- may not scale effectively within our eventual budget.

One clever way to sidestep some of those small budget concerns may be to attach to larger international conferences (e.g., Society for Neuroscience) with satellite conferences. Since so many of us will be there already, this could ease costs. There is also the opportunity / possibility of deliberately making some of these satellites topically distinct yet complimentary to the main theme of the meeting, which would hopefully help expose more people to new fields and inspire interdisciplinary collaboration (e.g., machine learning / AI satellite at SFN). 

Another intriguing idea that was floated is -- instead of a large meeting of many researchers of all career advancement -- to go with the dark smokey room of secret decision making (I'm thinking the X-files here). Silliness aside, the idea is that five of the worlds most powerful and well-funded researchers may be more effective for the greater collective than the greater collective itself. Perhaps some benevolent dictator or trickle down hopes and dreams here. 

Another key idea that landed strongly for me is the idea that a key limitation / gap is the absence of suitable incentive structure for research and development that genuinely fills that gap between natural and artificial intelligence. The idea would be for ARCANI to provide such an incentive structure by investing / funding a sort of competition in which teams compete not to create the best robot or AI overall (in an engineering optimality sense), but rather compete to create the most biologically compatible / plausible agent. I suppose I'm thinking something along the lines of this: 

https://www.brain-score.org/

**Comment by ABBarron** [comment]
ensuring equity of involvement.  It's really important in invitations to ask if people need any help to come.  That should be gender neutral but women can be disproportionately impacted by social constraints. ECR women may not feel able to ask for support and it is easy to offer an extra hotel room or whatever might help.
